src/app/server/Mdns.cpp

```diff
@@ -247,22 +242,27 @@ CHIP_ERROR Advertise(bool commissionableNode)
 /// (Re-)starts the minmdns server
 void StartServer()
 {
-    ChipLogProgress(Discovery, "Start dns-sd server");
+    StartServer(false, false);
+}
+
+void StartServer(bool commissioningMode, bool additionalCommissioning)
```

Comment:
Again, the callsites are pretty inscrutable.....

Reply:
replaced with bitmaps